docs/graphql/core/deployment/deployment-guides/heroku.rst

```diff
@@ -65,20 +65,20 @@ Step 2: Create an app with **--stack=container**
 ************************************************
 
 Use the `Heroku CLI <https://devcenter.heroku.com/articles/heroku-cli>`__ to create a new Heroku app from inside the cloned directory. Let's call
-the app ``graphql-on-postgres``.
+the app ``todo-hasura1``.
 
 .. code-block:: bash
 
   # Replace graphql-on-postgres with whatever you'd like your app to be called
```

Comment:
Here `graphql-on-postgres` should also be replaced with the new app name

Reply:
Thanks for pointing that out